### DISCUSSION TITLE
frontmatter key for know how outdated a converted HTML document is in es locale

### DISCUSSION THREAD

**Discussion by Graywolf9** [mdn/mdn-community#173] [Localization] [answered]
This space is for set a name for a frontmatter key to identified the `es` files with 2-5 years of outdated content this due to the es-team use the html format to know what are the most outdated content to update

**Comment by Graywolf9** [comment, marked as answer]
Not nessesary anymore

**Comment by pransh15** [comment]
Closing this as resolved. ✅
